packages/terra-search-field/src/SearchField.jsx

```diff
@@ -122,7 +127,41 @@ class SearchField extends React.Component {
     this.clearSearchTimeout();
   }
 
+  focus() {
+    this.currentInput.current.focus();
+  }
+
+  handleClear(event) {
+    // Clear uncontrolled input field
+    if (this.props.defaultValue && this.previouslyFocusedInput) {
+      this.previouslyFocusedInput.value = '';
+    }
+
+    // Pass along changes to consuming components using associated props
+    if (this.props.onChange) {
+      this.props.onChange(event, '');
+    }
+
+    if (this.props.onInvalidSearch) {
+      this.props.onInvalidSearch('');
+    }
+    this.setState({ searchText: '' });
+
+    // Set focus back on previous focus after clicking / pressing clear
+    if (this.previouslyFocusedInput) {
+      this.previouslyFocusedInput.focus();
+    }
+  }
+
+  handleInputFocus(event) {
+    if (this.previouslyFocusedInput !== event.currentTarget) {
+      this.previouslyFocusedInput = event.currentTarget;
+    }
+  }
+
   handleTextChange(event) {
+    this.handleInputFocus(event);
```

Comment:
Why are we doing this here instead of using a ref?

Reply:
I'm honestly not super familiar with the actual use of refs yet. I had attempted it, but I had some trouble getting it to work as intended in this case. If you can provide some guidance on how to incorporate it here, it would be appreciated!

Reply:
So it looks like the Input component forces the refCallback to be a function that will accept the dom node as a parameter and does not allow the new createRef syntax. But this way will be easier to still honor the inputRefCallback prop. You'd create a function like 
```
setInputRef(node) {
    this.inputRef = node;
    if (this.props.inputRefCallback) {
        this.props.inputRefCallback(node);
    }
}
```

Then on the input you would use that for the refCallback prop
`refCallback={this.setInputRef}`
Then you could use `this.inputRef` anywhere you'd need access to the dom node.

Reply:
We'd likely want to update Input so that the createRef syntax can be used, but that's probably out of scope of this PR

Reply:
Updated: [3c644fc](https://github.com/cerner/terra-core/pull/2317/commits/3c644fc526ca5b970747607ce166f4e43ecc93cf). TY for this! I like this solution much better.